common/src/main/res/drawable/ic_popup_close.xml

```diff
@@ -17,6 +17,5 @@
 <vector android:height="30dp"
     android:viewportHeight="30" android:viewportWidth="30"
     android:width="30dp" xmlns:android="http://schemas.android.com/apk/res/android">
-  <path android:fillColor="#F3F5F6" android:pathData="M15,30C17.049,30 18.9779,29.6078 20.7868,28.8235C22.5956,28.0392 24.1912,26.9559 25.5735,25.5735C26.9559,24.1912 28.0392,22.5956 28.8234,20.7868C29.6079,18.9779 30.0001,17.049 30.0001,15C30.0001,12.951 29.6079,11.0221 28.8234,9.2132C28.0392,7.4044 26.9559,5.8088 25.5735,4.4265C24.1912,3.0441 22.5931,1.9608 20.7794,1.1765C18.9657,0.3922 17.0343,0 14.9853,0C12.9363,0 11.0073,0.3922 9.1985,1.1765C7.3897,1.9608 5.7966,3.0441 4.4191,4.4265C3.0417,5.8088 1.9608,7.4044 1.1765,9.2132C0.3922,11.0221 0,12.951 0,15C0,17.049 0.3922,18.9779 1.1765,20.7868C1.9608,22.5956 3.0441,24.1912 4.4265,25.5735C5.8088,26.9559 7.4044,28.0392 9.2133,28.8235C11.0221,29.6078 12.951,30 15,30Z"/>
-  <path android:fillColor="#6D7885" android:pathData="M9.8236,21.4118C9.4902,21.4118 9.2059,21.2917 8.9706,21.0515C8.7353,20.8113 8.6177,20.5196 8.6177,20.1765C8.6177,19.8432 8.7353,19.5588 8.9706,19.3236L13.2794,15.0147L8.9706,10.7059C8.7353,10.4706 8.6177,10.1863 8.6177,9.853C8.6177,9.5098 8.7353,9.2231 8.9706,8.9927C9.2059,8.7623 9.4902,8.6471 9.8236,8.6471C10.1765,8.6471 10.4608,8.7647 10.6765,9L15,13.3089L19.3677,8.9853C19.603,8.7402 19.8824,8.6177 20.2059,8.6177C20.5392,8.6177 20.8236,8.7353 21.0589,8.9706C21.2942,9.2059 21.4118,9.4902 21.4118,9.8236C21.4118,10.1471 21.2942,10.4363 21.0589,10.6912L16.7353,15.0147L21.0442,19.3088C21.2795,19.5638 21.3971,19.853 21.3971,20.1765C21.3971,20.5196 21.2795,20.8113 21.0442,21.0515C20.8089,21.2917 20.5196,21.4118 20.1765,21.4118C19.853,21.4118 19.5638,21.2892 19.3089,21.0442L15,16.7353L10.7206,21.0442C10.4755,21.2892 10.1765,21.4118 9.8236,21.4118Z"/>
+  <path android:fillColor="@color/light_gray_black" android:pathData="M9.8236,21.4118C9.4902,21.4118 9.2059,21.2917 8.9706,21.0515C8.7353,20.8113 8.6177,20.5196 8.6177,20.1765C8.6177,19.8432 8.7353,19.5588 8.9706,19.3236L13.2794,15.0147L8.9706,10.7059C8.7353,10.4706 8.6177,10.1863 8.6177,9.853C8.6177,9.5098 8.7353,9.2231 8.9706,8.9927C9.2059,8.7623 9.4902,8.6471 9.8236,8.6471C10.1765,8.6471 10.4608,8.7647 10.6765,9L15,13.3089L19.3677,8.9853C19.603,8.7402 19.8824,8.6177 20.2059,8.6177C20.5392,8.6177 20.8236,8.7353 21.0589,8.9706C21.2942,9.2059 21.4118,9.4902 21.4118,9.8236C21.4118,10.1471 21.2942,10.4363 21.0589,10.6912L16.7353,15.0147L21.0442,19.3088C21.2795,19.5638 21.3971,19.853 21.3971,20.1765C21.3971,20.5196 21.2795,20.8113 21.0442,21.0515C20.8089,21.2917 20.5196,21.4118 20.1765,21.4118C19.853,21.4118 19.5638,21.2892 19.3089,21.0442L15,16.7353L10.7206,21.0442C10.4755,21.2892 10.1765,21.4118 9.8236,21.4118Z"/>
```

Comment:
This removes the background behind the X button and changes the color of the X to match most recent designs.